pkg/scheduler/framework/types.go

```diff
@@ -409,10 +409,6 @@ type NodeInfo struct {
 	// state information.
 	ImageStates map[string]*ImageStateSummary
 
-	// PVCRefCounts contains a mapping of PVC names to the number of pods on the node using it.
-	// Keys are in the format "namespace/name".
-	PVCRefCounts map[string]int
```

Comment:
why are you removing these instead of reusing this information in Prefilter?

Reply:
I thought I could rebuild the cache from scratch (but I see that's inefficient https://github.com/kubernetes/kubernetes/pull/114051#issuecomment-1322626693).

My cache kept references to the conflicting `*v1.Pod`s because I thought I'd need the conflicting pod's priority to determine which status to return. Since that isn't necessary, I think I can use this cache.

Reply:
I think you can still make it work with just integers.

This can store how many pods reference the PVC.

Then, in the state data, when you remove a pod, you can substract the number of conflicts that the pod has.